code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -139,61 +139,65 @@
 
 /obj/effect/alien/resin/special/pylon/endgame
 	cover_range = WEED_RANGE_CORE
+	protection_level = TURF_PROTECTION_OB
```

Comment:
I do like the idea of this only having OB protection when there's a hatchery under this pylon.

1.  Just unspecify this line
2. In `/datum/hivebuff/game_ender_caste/on_engage(obj/effect/alien/resin/special/pylon/purchased_pylon)` set `purchased_pylon.protection_level = TURF_PROTECTION_OB`
3. In `/obj/effect/alien/resin/king_cocoon/Destroy()` replace `GLOB.hive_datum[hive_number].has_hatchery = FALSE` with: 
```
	var/datum/hive_status/hive = GLOB.hive_datum[hive_number]
	hive.has_hatchery = FALSE
	for(var/obj/effect/alien/resin/special/pylon/pylon as anything in hive.active_endgame_pylons)
		pylon.protection_level = initial(pylon.protection_level)
```
4. In `/obj/effect/alien/resin/king_cocoon/proc/hatch_king()` have the same protection level reset code.